indexing-service/src/main/java/io/druid/indexing/overlord/ThreadPoolTaskRunner.java

```diff
@@ -92,12 +105,18 @@ public ThreadPoolTaskRunner(
     return ImmutableList.of();
   }
 
-  private static ListeningExecutorService buildExecutorService(int priority)
```

Comment:
@drcrallen I changed this from `static` to non-static, it that alright ? you added this method [here](https://github.com/druid-io/druid/pull/984/files#diff-4bb98794072094a98cddeed6166ea13fR95) but I do not find any static usages for it.